### ISSUE TITLE
Issues with meteor.local when developing on mobile

### ISSUE THREAD

**Issue by rclai** [jedwards1211/meteor-webpack-react#109]
I'm trying to get this to work with mobile and for some reason `meteor.local:9090/assets/client.bundle.js` is not getting me the actual bundle script but is rather giving me the entire HTML mark-up of a Meteor app, which is not right. I had to hard-code the script embed in `loadClientBundle.html` to the DDP server's host address (i.e. 192.168.1.123:9090), in order for it to work properly.

**Comment by jedwards1211** [comment]
I've never heard of this `meteor.local` thing being possible, is there documentation for it anywhere?  I'd have to know how exactly it works to offer any support.  Is this for Cordova?  Whenever I've tested on mobile (though not Cordova) I type my dev machine's IP address into the URL bar (e.g. `http://192.168.1.123:3000`) and that works.

**Comment by rclai** [comment]
Yes, when you fire up a Meteor Cordova app, the host designated is `meteor.local`. We'd have to dig into the code to see how that works.

**Comment by rclai** [comment]
Check [this](https://github.com/meteor/meteor/blob/dc3cd6eb92f2bdd1bb44000cdd6abd1e5d0285b1/tools/cordova/builder.js#L127) out.

**Comment by rclai** [comment]
I guess we can detect `meteor.local`, but then we'd have to somehow get the host of the DDP server somehow.
